src/Components/AboutPage/index.jsx

```diff
@@ -0,0 +1,30 @@
+const AboutPage = () => {
+    return (
+        <>
+            <p>
+                Welcome to Caterpillar OS. This is a browser-based operating system that the team has built during our final week of iO Academy. The OS was primarily built on the front-end using React-based JavaScript and styled using Tailwind CSS, however we have included some back-end features using PHP to perform tasks such as recognizing and storing login information in addition to extracting data from an API we have built using an SQL database.
```

Comment:
Put the 'Welcome to Caterpillar OS' in a heading tag? and then a div around the whole section of the heading and p tags